### DISCUSSION TITLE
"Breakdown of the Newton–Einstein Standard Gravity at Low Acceleration in Internal Dynamics of Wide Binary Stars"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#122] [Gravitation Models]
*"Breakdown of the Newton–Einstein Standard Gravity at Low Acceleration in Internal Dynamics of Wide Binary Stars"*
Open Access
[https://iopscience.iop.org/article/10.3847/1538-4357/ace101](https://iopscience.iop.org/article/10.3847/1538-4357/ace101)
Kyu-Hyun Chae 2023 ApJ 952 128
doi 10.3847/1538-4357/ace101

**Comment by khuramonline** [comment]
**First thing:** Premise is false that Newton Gravity fails at low accelerations. 

**Second Thing:** Wide binaries are not conclusive. Even if for "deep MOND regime (low acceleration) wide binaries tally with MOND that has the reason that rest of the galaxy works like hypothetical dark matter. All the wide binaries are surrounded by the real stuff that is counterpart of hypothetical dark matter.

**Comment by HanDeBruijn** [comment]
Question: __what is a wide binary__ in the first place?
Learning from _[How far can a pair of stars be separated and still maintain a stable orbit around each other?](https://www.astronomy.com/science/how-far-can-a-pair-of-stars-be-separated-and-still-maintain-a-stable-orbit-around-each-other/)_ , concise information is obtained:
> An ultrawide binary with a separation of 0.5 parsec (1.6 light-years) is statistically likely to break up within just 100 million years, while a slightly less extreme binary with separation around 0.1 pc (0.3 light-years) may survive for more than **1 billion years**.

Let's dig into the [main paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace101) for this. kAU units are used instead of parsecs. The definitions are: $1 AU = (149,597,870,700) m$ and $1 pc = (648,000/\pi) AU$. In the paper, wide binaties are restricted to those with a separation $4 \lt s \lt 30$ kAU between the two stars. This is in agreement with the value from our first article: $0.1 pc = 64.8/\pi \approx 20.6$ kAU.

Quoting from the Abstract:
>A gravitational anomaly parameter $\delta_{obs−newt}$ between the observed acceleration at $g_N$ and the Newtonian prediction is measured to be: $\delta_{obs−newt} = 0.034 \pm 0.007$ and $0.109 \pm 0.013$ at $g_N ≈ 10^{−8.91}$ and $10^{−10.15} m s^{−2}$ [ .. ] At $g_N = 10^{−10.15} m s^{−2}$, the observed to Newton-predicted acceleration ratio is $g_{obs}/g_{pred}=10^{\sqrt{2}\delta_{obs-newt}}=1.43\pm 0.06$ .

As a humble theorist **I understand nothing of the above**. Why for example is there a difference between "_newt_" and "_pred_" ?
These things are **not clarified** further on in the paper, where it is read that: 
>Because $\delta_{obs-newt}$ is an orthogonal difference in log space, the ratio of the observed kinematic acceleration $g_{obs}$ to the Newtonian predicted kinematic acceleration $g_{pred}$ is given by

$$
\large \frac{g_{obs}}{g_{pred}} = 10^{\sqrt{2}\delta_{obs-newt}} \quad (22)
$$

However, some equations from the [paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace101) are understandable even for a theorist.
Newtonian acceleration:

$$
g_N(r) = \frac{GM_{tot}}{r^2} \quad (14)
$$

The [standard gravitational parameter](https://en.wikipedia.org/wiki/Standard_gravitational_parameter) of the sun is: $\mu = 1.32712440042 \times 10^{20} \mbox{ } m^3/s^2$ . Let's calculate the separation $s=r$ between two stars like the sun to see if s = 0.1 pc makes sense.

$$
g_N(low) = \frac{2GM(sun)}{r^2} \quad \Longrightarrow \quad r = \sqrt{\frac{2GM(sun)}{g_N(low)}}
$$

```
GM := 1.32712440042*10^(20);
g_N := 10^(-10.15);
pc := 3.08567758*10^(16);
r := sqrt(2*GM/g_N)/pc;

                          r := 0.06275094490 # parsec
```
We conclude that 0.1 pc is a reasonable separation.
Kinematic acceleration:

$$
g(r) = \frac{v^2(r)}{r} \quad (15)
$$

It follows that

$$
v(r) = \sqrt{g(r)\cdot r}
$$

with $v=$ tangential velocity, $r=$ related to separation $s$ between the stars = distance to the center of mass (?), $g=$ acceleration.
Switching to our own theory, remember that [MOND = UAC](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) and read about [A Basic Equation](http://www.alternatievewiskunde.nl/cosmology/simpel.htm). The latter is repeated here for convenience:

$$
\large e^{-H_i/c_0 \mbox{ }  r} \mbox{ } \vec{a}_T = \vec{a}_t - \frac{1}{2} H_i \vec{v}_t
$$

Symbols: $H_i=$ (intrinsic) Hubble parameter, $c_0=$ lightspeed, $r=$ distance to earthly observer (different from the paper), $a_T=$ acceleration in orbital (Newtonian) timeframe, $a_t=$ acceleration in atomic (Observed) timeframe, $v_t=$ velocity in atomic (Observed) timeframe.
According to the [paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace101), distances to the binaries are restricted to the range (80 pc - 200 pc). This ensures that the exponent in $e^{-H_i/c_0 \mbox{ } r}$ is near zero, resulting in a factor nearly equal to one.
```
c := 299792458;
Mpc := 3.08567758*10^22;
H := 70;
H := H*1000/Mpc;
r := 200*Mpc/10^6;
exp(-H/c*r);

           0.9999999533
```
Replace all vectors by scalars and substitute $a_T \to g_N(r)$ , $a_t \to g(r)$ , then we have

$$
g(r) - g_N(r) = \frac{1}{2} H_i v(r) \quad \Longrightarrow \quad \large \frac{g-g_N}{g_N} \approx \frac{1}{2} H_i \sqrt{\frac{r}{g}}
$$

The square of the relative "error" in Newtonian dynamics is directly proportional to the wideness of the binary and inversely proportional to the acceleration, entirely **due to an effect of our [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs)**. Finally we have

$$
\large \frac{g_{obs}}{g_{pred}} \approx 1+\frac{1}{2} H_i \sqrt{\frac{r}{g}} \quad (?)
$$

```
kAU := 149597871 * 10^6; # meters
pc := 3.08567758 * 10^(16); # meters
# Stars separation
r := 0.1*pc;
# Number of seconds in a year
year := 31556926;
# Intrinsic Hubble parameter
H_i := 2/(10^9*year);
# Low Acceleration
g := 10^(−10.15);
# Tangential velocity
v := sqrt(g*r);
# Additional UAC term
result := 1+H_i/2*v/g;

                        result := 1.000209209 # =/= 1.43
```
**How does this result compare with values in the [paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace101) ?**

**Comment by RedshiftDrift** [comment]
"Ask Ethan: Do binary stars prove modified gravity?"
[https://bigthink.com/starts-with-a-bang/binary-stars-prove-modified-gravity/](https://bigthink.com/starts-with-a-bang/binary-stars-prove-modified-gravity/)

**Reply by khuramonline** [reply]
Ethan Siegal: Official stance is always correct. Einstein's Theory is always victorious. Criticism on official science is always wrong.

Anyhow, he has one correct point. Wide binaries are located within the galaxy.

> As we go to very wide separations, even if your binary stars remain gravitationally bound, how are their accelerations influenced by both the presence of potentially massive Oort clouds (especially if the stars are young and massive) and by the background density of dark-and-normal matter that permeates the Milky Way? The assumption is “not at all,” but that may not be correct.

On the whole, he himself is doing bad science. Many years before I just first time interacted with him on his twitter post. Off course I just somewhat criticized his points. Then my twitter account was permanently suspended with no right to appeal.

Forbes, Twitter (of that time) and Medium are / were running on same policy. To keep this clown science communicator safe from any criticism.

**Reply by budrap00** [reply]
@khuramonline I've never used the Musk melon formerly known as Twitter but I criticize Siegel on Medium often. I take care to avoid _ad hominem_ remarks and I have never had a problem. I understand your reason for calling him a "clown" but if you use that kind of language in your criticism on well-run public forums you will get banned. See my critique here for example:

https://medium.com/starts-with-a-bang/jwst-at-last-answers-our-biggest-cosmic-questions-5e3881f33d6

**Reply by khuramonline** [reply]
I had written on his tweet something like ... Your theories suggest that you know how many angels are dancing on a pin nib. I had used these types of words. I also used words that included medieval ages because that problem of angels dancing related to that era. When i commented that, at that time Elon Musk was not the owner of twitter.

He could tell and he always keep on telling that he knows exact number of angels dancing on a singularity point. Off course he is not 'cl**n' by his profile picture. 

If I used bad words, they banned me. But I stand by my words. He and his directors have made science into a proper cult.

**Reply by khuramonline** [reply]
Have seen your comments there. Good to see that you also had same complaint against him as I had. Your words were humble but you also used word cult for him. My harsh words can have other reasons ... like my fight experience with other cults.

**Comment by sahil5d** [comment]
If [wide binaries are explained by MOND], a problem arises, which is the implication that MOND accelerations are vector addable, which results in a discrepancy. For example, if Star A in a wide binary were accelerating at the mond-acceleration to Star B, while also accelerating at the mond-acceleration to the center of the galaxy (treating the galaxy as a unit), that would mean Star A's total, instant acceleration is the sum of those vectors. So [wide binaries are explained by MOND], to be compatible with Star A's total acceleration, would require [MOND accelerations are addable].

But if they're addable, then why not start calculating individual mond-accelerations from Star A to the trillion other stars in the galaxy, and sum those vectors? Because that value would conflict with (it would be ~ a million times larger) the mond-acceleration to the galaxy center (treating the galaxy as a unit).

So, accepting [MOND explains wide binaries] requires [MOND accelerations are addable], which allows a procedure to calculate total acceleration which leads to a false value. Therefore [MOND explains wide binaries] has an issue.

Does this make sense?

**Reply by HanDeBruijn** [reply]
>a problem arises, which is the implication that MOND accelerations are vector addable
Does this make sense?

Yes, I think that it makes sense.

Let's quote from the Wide Binary [paper](https://iopscience.iop.org/article/10.3847/1538-4357/ace101) at 5.3, emphasis by me:
>Finally, the clear gravitational anomaly suggests that MOND is realized by _modified gravity rather than modified inertia_. [ .. ] modified gravity seems favored, although the work has not directly distinguished between modified gravity and modified inertia.

Maybe this must be interpreted as: modifying $\vec{F} = m\vec{a}$ is different from modifying Newton's law of gravitation.

### Variable Mass theory

I have another proposal - no surprise: it's (my [version](http://www.alternatievewiskunde.nl/cosmology/wild.htm) of) the Variable Mass (VM) theory.
Instead of Milgrom's ad-hoc assumption $F = m a^2/a_0$ sort of a MOND mechanism rather should be **derived** from more foundational and generally accepted principles.
- The frequency $(1/dt)$ of an electron [in an atom](https://en.wikipedia.org/wiki/Rydberg_constant) is proportional to mass $m \\Longrightarrow$ atomic time $t$
- The frequency $(1/dT)$ of a body [in an orbit](https://en.wikipedia.org/wiki/Orbital_mechanics#Orbital_period) is proportional to the square root of (gravitating) mass $m \Longrightarrow$ orbital time $T$

A quotient of the two frequencies immediately leads to a very fundamental theorem of the VM theory.
The two clocks are _synchronized_ here and now at $t=T=0$ such that $dt_0=dT_0$ .

$$
\frac{dT}{dt} = \frac{dT/dT_0}{dt/dt_0} = \frac{\sqrt{m_0/m}}{m_0/m} = \sqrt{m/m_0}
$$

According to the Variable Mass Theory [video](https://www.youtube.com/watch?v=T7jVviVBzKs) at 1:50 Narlikar's law is formulated as $m = a t^2$ . Within our own version of the same theory, it must be reformulated a bit more precisely:

$$
\large \frac{m}{m_0} = (1+T/A)^2
$$

with $A=$ Age and such that $m=m_0$ for $T=0$ . It follows that

$$
\frac{dT}{dt} = 1+T/A \quad \mbox{and} \quad \frac{dt}{dT} = \frac{1}{1+T/A}
$$

There is a difference between orbital velocity $v_T$ as calculated with Newton and atomic velocity $v_t$ as observed with light ($r=$ distance):

$$
\vec{v_t} = \frac{d\vec{r}}{dt} = \frac{d\vec{r}}{dT}\frac{dT}{dt} = \vec{v_T}\sqrt{m/m_0} = (1+T/A)\vec{v_T}
$$

The difference becomes even more pronounced with acceleration (by repeated differentiation):

$$
\vec{a}_T = \frac{d^2\vec{r}}{dT^2} = \frac{d}{dT}\left(\frac{d\vec{r}}{dt}\frac{dt}{dT}\right) = \frac{d}{dT}\left(\frac{d\vec{r}}{dt}\right)\frac{dt}{dT} + \frac{d\vec{r}}{dt}\frac{d}{dT}\left(\frac{dt}{dT}\right) = \frac{d^2\vec{r}}{dt^2}\left(\frac{dt}{dT}\right)^2 + \frac{d\vec{r}}{dt}\frac{d^2t}{dT^2}
$$

where

$$
\frac{dt}{dT} = \frac{1}{1+T/A} \quad \Longrightarrow \quad \frac{d^2t}{dT^2} = -\frac{1/A}{(1+T/A)^2}
$$

and so

$$
\vec{a}_T = \frac{d^2\vec{r}}{dt^2}\left(\frac{1}{1+T/A}\right)^2 - \frac{d\vec{r}}{dt}\frac{1/A}{(1+T/A)^2} = \frac{\vec{a}_t - \vec{v}_t/A}{(1+T/A)^2}
$$

It is thus seen that even for $T=0$ - that is _here and now_ - there is an **anomalous** component of the acceleration, which makes a difference for the two timeframes (orbital vs. atomic):

$$
\vec{a}_T = \vec{a}_t - \vec{v}_t/A \quad \mbox{or} \quad \vec{a}_t = \vec{a}_T + \vec{v}_T/A
$$

### Galaxy rotation curves
Contrary to what one would expect, Mordegan's _acceleration_ in MOND is analogous to **velocity** in the VM theory. It is assumed that (elementary rest) mass is varying across the galaxy, in such a way that new mass is _created_ in the center and becomes more aged when spiralling in outward direction $r$ . As a first attempt to mimick the true [Galaxy rotation curve](https://en.wikipedia.org/wiki/Galaxy_rotation_curve), the following _hypothesis_ has been tried: (elementary rest) mass is proportional to the distance from the [center](http://www.alternatievewiskunde.nl/melkweg/zwartgat.htm) of the galaxy. Formally: $m \sim r$ , simply a _linear_ relationship to begin with. Let $R=$ radius of the galaxy, then

$$
|\vec{v}_t| = |\vec{v}_T|\sqrt{m/m_0} = |\vec{v}_T|\sqrt{r/R} \quad \Longrightarrow \quad v_t = \sqrt{r/R}.v_T
$$

According to Newtonian mechanics, for the circular orbit of a star around a gravitating body with mass $M$ :

$$
\frac{GMm}{r^2} = m\frac{v_T^2}{r} \quad \Longrightarrow \quad v_T = \sqrt{GM/r} \quad \Longrightarrow \quad v_t = \sqrt{r/R}.\sqrt{GM/r} = \sqrt{GM/R}
$$

which is indeed a **flat rotation curve**, as observed in the atomic time frame with our telescopes.
Without MOND, without Dark Matter.

### Appendix

The Variable Mass Theory has originally been devised for the purpose to have an explanation for the cosmic redshift.
With $\lambda=$ wavelength of light, the redshift $z$ is defined as

$$
z = \frac{\lambda - \lambda_0}{\lambda_0} \quad \Longrightarrow \quad 1+z = \frac{\lambda}{\lambda_0} = \frac{c.dt_0}{c.dt} = \frac{m}{m_0} = (1+T/A)^2
$$

With help of the previous, relationships between orbital time $T$ and atomic time $t$ are derived.

$$
\frac{dt}{dT} = \frac{1}{1+T/A} \quad \Longrightarrow \quad t = \int\frac{dT}{1+T/A} = A.\ln(1+T/A) \quad \Longrightarrow \quad 1+T/A = e^{t/A}
$$

Combining with the redshift gives for the cosmic distance $D$

$$
D = c.t = c.A.\ln(1+T/A) = c(A/2)\ln([1+T/A]^2) = c(A/2)\ln(1+z)
$$

A well known formula is thus re-established.

$$
\large D = c/H.\ln(1+z)
$$

Provided that the Hubble parameter is not one divided by Age, but **twice** that value: $H=2/A$ (Narlikar).
At last, the Variable Mass can be expressed in both orbital and atomic time with the Hubble:

$$
\frac{m}{m_0} = \left(1+\frac{1}{2}H.T\right)^2 = \large e^{H.t}
$$

**Reply by sahil5d** [reply]
> The frequency of an electron in an atom is proportional to mass m => atomic time t

Based on what?

> A quotient of the two frequencies immediately leads to a very fundamental theorem of the VM theory.

What is the basis for making this quotient?

> new mass is created in the center and becomes more aged when spiralling in outward direction r

According to what?

I'm not convinced your theory is offering any clarity on the known v^4 ~ M galactic relation.

___

The improvement to MOND, in my opinion, should connect MOND's galactic acceleration contribution to local stellar, planetary acceleration contributions.

For example, the ISS has an instant acceleration
[~ 9 m/s^2 in the dir of the Earth] + [~ 6 * 10^-3 m/s^2 in the dir of the Sun] + [~ 2 * 10^-10 m/s^2 in the dir of galactic center], a vector sum.
(The ~ 2 * 10^-10 m/s^2 is > a0, so it's not an exemplar of apply-MOND-when-g<a0, but the point is the **total acceleration** of an object orbiting a planet is a sum of components which weren't all calculated the same way. That needs justification.)

The improvement to MOND should output the total acceleration of a particle given a 3D point cloud of a galaxy (a list of points with their mass, position, velocity, and for test, acceleration).

**Reply by HanDeBruijn** [reply]
>I'm not convinced your theory is offering any clarity on the known v^4 ~ M galactic relation.

See [MOND &sube; VMT](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) :

$$
\left(v_t^2\right)^2\mbox{ (VMT)} = v^4\mbox{ (MOND)} \quad \Longrightarrow \quad  \left(\frac{GM}{R}\right)^2 = GM\mbox{ }a_0 = v^4\mbox{ (MOND)}
$$

As for the rest of your questions, I have made adjustments in place.

**Reply by sahil5d** [reply]
You referenced Rydberg's constant... how does that prove "The frequency of an electron in an atom is proportional to mass m" ?

> A quotient of the two frequencies immediately leads to a very fundamental theorem of the VM theory.

Why make this quotient?

> new mass is created in the center and becomes more aged when spiralling in outward direction r

What is the evidence?

___

An improvement to MOND will produce the total acceleration on a particle.

Here is one attempt, which is a program that runs on every particle.
```
const r_cutoff = 10^15 meters    // ~ 10,000 AU, this is a guess

function calc_accel_for_particle() {
    a_classic = 0
    a_far = 0

    for each mass:               // each mass in the 3D point cloud
        a_classic += Gm/r^2

    for each mass farther than r_cutoff:
        a_far += Gm/r^2
    a_far = sqrt(a_far * c^2/87Gly)

    a_total = a_classic + a_far

    return a_total
}
```
but it is not elegant as it doesn't explain why there's an r_cutoff, why it has a certain value and can be no other value, and why there's a sum of components which weren't all calculated the same way.

**Reply by sahil5d** [reply]
The r_cutoff, *beyond which* a separate acceleration program applies, is similar to the current understanding of the nuclear force, which has an r_cutoff of ~ 2.5 * 10^-15 m, *within which* a separate force program applies.

The total charge-related force on a proton considering all particles in a 3D point cloud is something like this:
```
const r_cutoff = 2.5 * 10^-15 meters    // https://en.wikipedia.org/wiki/Nuclear_force

function calc_force_on_proton() {
    f_near = 0
    f_classic = 0

    f_near = specialfx(all particles within r_cutoff)    // don't know the special function, but f_near is attractive

    for each particle:
        f_classic += k(q_proton)(q_particle)/r^2

    f_total = f_near + f_classic

    return f_total
}
```

Does anyone have a better way to calculate the total charge-related force on one proton given a 3D point cloud?

Also, an explanation for [why Modified Coulombian (or Coulombic) Dynamics kicks in within ~ 2.5 * 10^-15 m] might open a door to [why MOND-redefined-for-total-acceleration-calculation kicks in beyond ~ 10^15 m].

**Reply by HanDeBruijn** [reply]
Our [Galaxy project](http://www.alternatievewiskunde.nl/melkweg/index.htm) and the [Continuing story](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm) of it has resulted in the following [(Delphi) Pascal](https://en.wikipedia.org/wiki/History_of_Delphi_(software)) code.
```
{
This software has been designed and it is
CopyLefted by Han de Bruijn:

                          (===)
                         @-O^O-@
                          #/_\#
                           ###

UNIFIED ALTERNATIVE COSMOLOGY GALAXY PROJECT
============================================
}
```
[Rest](http://www.alternatievewiskunde.nl/melkweg/project9.txt) deleted, for the sake of saving bandwidth.

**Reply by RedshiftDrift** [reply]
> accepting [MOND explains wide binaries] requires [MOND accelerations are addable], which allows a procedure to calculate total acceleration which leads to a false value. Therefore [MOND explains wide binaries] has an issue.
> Does this make sense?

@sahil5d your argument makes sense for [MOND as modified inertia] = [ $\vec{F} = m \vec{a} \mu(a/a_0)$ ] !
Since the "interpolating" function $\mu(a/a_0)$ is non linear, the order of the summation is important.

If a star $i$ produces an acceleration $a_i$ on star $A$, it is not clear how to do the summation to determine if star $A$ is in the MOND regime or not.

In other words, the force on star A obtained by adding the individual MOND contribution of each other star $i$ (including star B) in the galaxy

$$
\vec{F}_A = m_A \sum_i \vec{a_i} \mu \left( \frac{a_i}{a_0} \right), \quad \mbox{where} \quad a_i=|\vec{a}_i|,
$$

is not the same as the force obtained by taking the vector sum on the entire galaxy as a unit

$$
\vec{F'}_A =  m_A  \left[ \sum_i \vec{a_i} \right] \ \mu \left( \frac{ \left| \sum_j \vec{a}_j \right|}{a_0} \right).
$$

I haven't read the "Wide Binary paper" in details, do they add star B separately from the galaxy?

If so, this would mean

$$
\vec{F''}_A = m_A \left[ \sum_i \vec{a_i} \right] \ \mu \left( \frac{ \left| \sum_j \vec{a}_j \right|}{a_0} \right) +  m_A \vec{a_B} \mu \left( \frac{a_B}{a_0} \right) \quad \mbox{ for} \quad i \ne B
$$

which would be ***a serious issue***.

For [MOND as modified gravity] they use the total potential, so that would be like calculating $\vec{F'}_A$...


P.S. This is the arbitrary part of MOND that I don't like, Milgrom *et al.* are just trying different equations that don't make physical sense, so they are free to "move the goal posts" until they find an equation that works.
For exemple, why not calculate

$$
\vec{F'''}_A = m_A \left[ \sum_i \vec{a_i} \right] \ \mu \left( \frac{ \sum_j \left| \vec{a}_j \right|}{a_0} \right).
$$

which would give another value!

**Comment by AngelBlanco19** [comment]
> "Breakdown of the Newton–Einstein Standard Gravity at Low Acceleration in Internal Dynamics of Wide Binary Stars"

How much longer will we need to continue with the same old stories, with a bigger lie than the previous one to cover it up?. Is it not already time to enter fully into the deep essence of the phenomenon?. 

I will try to summarise the essence as much as possible. From the fact that an apple falls on the head of someone sitting below, IT CANNOT BE INFERRED IN ANY WAY that the Moon has to fall in someone's backyard. In fact it is moving away, just as the Earth-Moon system is moving away from the Sun. Bodies without Gravity are doomed to fall on those with Gravity and celestial bodies with Gravity all move away from each other. And not only that, all celestial bodies with Gravity (except naked stars or misnamed "black holes") and their respective finite, determined and countable gravitational fields are Flat Spheres by definition, which are in a constant eternal process of growth and expansion. The older a celestial body with Gravity is, the faster it grows and expands along with its finite, determined and countable gravitational field, until it reaches a certain physical limit.

Gravity is the second most important physical process of infinite and eternal Matter. The finiteness of Matter inevitably entails postulating the existence of non-existence, which is a logical aberration. The main function of the physical process of Gravity is the hot, constant and eternal nuclear synthesis of chemical substances, for which it must first transform Photons, creating new elementary particles of Matter (28 in total) and as waste, Gravitons are formed. So we have, that among the 28 different elementary particles of Matter, we have 4 bosons, and of them the two fundamental elements of every gravitational field – the Universal boson, which is the building block of EVERYTHING - the so–called Photon (95-97% of Matter), and the “heavier” boson, the waste or rest of the hot nuclear synthesis - the so-called Graviton. Without Gravitons there is no gravitational field possible. The nucleus of Gravitons is supported by the Photons that surround it, thus creating the necessary support for the movement at infinity, to be able to rotate (translation moving infinity is impossible). They therefore rotate in opposite directions. Thus celestial bodies with Gravity or without can move in curvilinear trajectories around another celestial body with older Gravity.

The rotation of the Graviton nucleus creates the necessary temperature, so that the Photon transformation occurs first and then the hot nuclear synthesis, with strong, not so strong and weak bonds between the new particles that are not bosons (there are 24 left for this therefore). Of course, the first atom that is synthesized is that of hydrogen. 6 photons are consumed for each hydrogen atom synthesized and a Graviton is formed (6 = 5 + 1). The used Photons are replenished immediately from the inner infinity – this is the cause of growth and expansion. The Matter is inexhaustible.

The physical process of Gravity has several side effects, and ONLY ONE of THEM is what “science” and therefore you call “gravity” – the “fall” into a gravitational field or the even more aberrant “attraction” between “masses”. Will there be any common sense left?.

The so-called ”fall“ or ”rise", depending on the substance in question, is nothing more than the tendency to translational movement (the tendency is towards where there is a greater frequency differential between Photons, depending on the characteristics of the atom in consideration), rotating along with the field, due to the frequency differential created between the elements of the field (Photons). The frequency differential is created by the different powers of infinity of the Photons that make up the field, product of the fact that said field is a FLAT SPHERE – this is another side effect, a unique and exclusive product of the physical process of Gravity. Each photon layer relies on neighboring layers for its rotation as a layer. This Flat Sphere is in permanent and eternal rotation, moved by a nucleus of Gravitons that rotates in the opposite direction and that both - nucleus and field, grow and expand constantly up to a certain and determined physical limit. It is understood, then, that the synthesized chemical substances are located within the field, as close as possible to the Graviton nucleus, which is the main source of heat of any celestial body with Gravity. The rotation speeds of each Photon layer were determined many years ago by Keppler's Constant (t^2/a^3). It is unique for each gravitational field and depends on its age – it varies gradually over time, as each field grows and expands. One gravitational field does not see another, unless they come into direct physical contact (example - when the Moon with its field, begins to leave the Earth's field).

A natural satellite will evolve into a planet, a planet will evolve into a star and a star will evolve into a naked star (misnamed “black holes”) – the last stage of its development and will remain in this cyclic state of explosions, diameter reduction and further growth, eternally. It simply cannot continue to grow infinitely, the force between Gravitons has a limit, which the centrifugal force, which they themselves create with their rotation, breaks, surpasses it. Among these normal stages of development there are all possible sub-variants.

**Reply by AngelBlanco19** [reply]
> The frequency differential is created by the different powers of infinity of the Photons that make up the field, product of the fact that said field is a FLAT SPHERE – this is another side effect, a unique and exclusive product of the physical process of Gravity.

Every celestial body with gravity (except naked stars or misnamed "black holes") and their respective finite, determined and countable gravitational fields are flat spheres by definition. The issue lies in the very definition of a straight plane – any one, whose radius of curvature at any point on its surface is equal to infinity. This is what happens with our planet – the radius of curvature at any point of its spherical surface is equal to infinity, and therefore it is a «flat sphere» or if you prefer, a «straight plane curved on itself». Look at my work: «The Gravitational Paradox and the Proposed Experiment to Demonstrate Gravitational Distortion of "Spacetime"» In English, Russian and Spanish versions, revised and expanded. 15 pages. 2 drawings.
[https://vixra.org/abs/2202.0042](https://vixra.org/abs/2202.0042)
